UPGRADING

```diff
@@ -280,6 +280,9 @@ PHP 7.1 UPGRADE NOTES
   . Timezone initialization failure from serialized data will now throw an
     instance of Error from __wakeup() or __set_state() instead of resulting in
     a fatal error.
+  . DateTime instances now support microseconds. Please note this can have impact
+    when comparing; eg. this would previously return TRUE, but now returns FALSE:
+     (new DateTime('first day of next month') == new DateTime('first day of next month'));
```

Comment:
> True, but if generating a DateTime takes roughly 4 μs (which it does for me), this happens ~ 250.000 times more often

That doesn't make that the developer's original assumption was correct. There was still a bug in their code.

People not understanding an issue should not prevent fixing bugs. It's in the same league as not knowing about DST and assuming that each day is 86400 seconds long.

I will add something to UPGRADING, but with a better wording, so will be closing this PR.